release_notes_generator/record/record_factory.py

```diff
@@ -41,17 +45,6 @@ class RecordFactory:
     def generate(
```

Comment:
You are missing a method doc for this function.

Reply:
Yes, I missed it. Fixed.